src/serlio/utils/MayaUtilities.cpp

```diff
@@ -1,7 +1,46 @@
 #include "utils/MayaUtilities.h"
 #include "utils/MELScriptBuilder.h"
 
+#include "maya/MStringResource.h"
+#include "maya/MStringResourceId.h"
+
+#include <map>
 #include <memory>
+#include <string>
+
+namespace {
+constexpr const wchar_t KEY_URL_SEPARATOR = L'=';
+const std::wstring INDIRECTION_URL = L"https://raw.githubusercontent.com/Esri/serlio/indirection_urls/urls.txt";
```

Comment:
Please rename the `indirection_urls` branch to `data`, we might use it for other files as well.

Reply:
done.